extensions/panache/mongodb-panache-kotlin/runtime/src/main/kotlin/io/quarkus/mongodb/panache/kotlin/reactive/ReactivePanacheMongoRepositoryBase.kt

```diff
@@ -0,0 +1,832 @@
+@file:Suppress("unused")
+
+package io.quarkus.mongodb.panache.kotlin.reactive
+
+import io.quarkus.mongodb.panache.kotlin.reactive.ReactivePanacheMongoEntityBase.Companion.operations
```

Comment:
It's not reusing the object.  it's just defining one instance of the operations class.  this simplifies the bytecode generation in that we can always delegate to the same place.  this way we have one set of code generating the necessary bytecode between the active record and repository approaches.  Otherwise we'd need two sets of code to generate that byte code or we'd have to paramterize the owner of that reference.  This way was just vastly simpler.  It's invisible to devs at that point.